telegram_export/formatters/htmltemplates.py

```diff
@@ -0,0 +1,150 @@
+column_width = 480
```

Comment:
Perhaps a comment about what this controls?

Reply:
Not totally necessary, just a thought. Of course, comments aren't the best solution as they can become outdated.

Reply:
> comments aren't the best solution as they can become outdated.

That's a poor argument. Maybe telegram-export could support `--formatter-params column_width=480`, or something like that. Having a standard way to define which parameters are valid could also help with error diagnostics, help or just listing options.